cookiecutter.json

```diff
@@ -2,7 +2,7 @@
     "pipeline_name": "NF-Something",
```

Comment:
If we want `nf-core` as a default, I think it should be here as part of the pipeline name. Then everything else follows on. I also think it's nice to have the full "name" of the pipeline include nf-core. So:

```json
"pipeline_name": "nf-core/example",
```